tools/test-tools/.eslintrc.js

```diff
@@ -5,13 +5,19 @@
 
 module.exports = {
     "extends": [
-        require.resolve("@fluidframework/eslint-config-fluid")
+        require.resolve("@fluidframework/eslint-config-fluid/recommended")
     ],
     "parserOptions": {
         project: ["./tsconfig.json"],
     },
+    "settings": {
+        "import/resolver": "node"
+      },
     "rules": {
         "@typescript-eslint/ban-ts-comment": "off",
+        "@typescript-eslint/no-unsafe-assignment": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
+        "import/no-nodejs-modules": "off",
```

Comment:
Prefer to configure this rule to only allow the node packages we are definitely using in the package.

Reply:
@chentong7 I think what Tyler meant was that it would be preferred to do something like [this](https://github.com/microsoft/FluidFramework/blob/main/packages/drivers/local-driver/.eslintrc.js#L15), that keeps the rule enabled but still lets certain specific packages (the ones we need in the *.ts files) be imported.

Can you put that change in another PR?